# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaManager.h
# Commit: 1ce0a5921f1c
# Full Hash: 1ce0a5921f1c6169de6e97da32e9801a0b65827b
# Author: Paul Adenot <paul@paul.cx>
# Date: 2020-03-27 21:52:07
# Regressor Bug: 1624322
# File Overlap Count: 1
# Description:
#   Bug 1624322 - Add the raw group ID to MediaDevice. r=jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D68082
# ==============================================================================

diff -r 6502d0dbe2be -r 1ce0a5921f1c dom/media/MediaManager.h
--- a/dom/media/MediaManager.h	Fri Mar 27 14:12:55 2020 +0000
+++ b/dom/media/MediaManager.h	Fri Mar 27 14:16:27 2020 +0000
@@ -75,11 +75,12 @@
               const nsString& aRawID = NS_LITERAL_STRING(""));
 
   MediaDevice(const RefPtr<MediaDevice>& aOther, const nsString& aID,
-              const nsString& aGroupID, const nsString& aRawID);
+              const nsString& aGroupID, const nsString& aRawID,
+              const nsString& aRawGroupID);
 
   MediaDevice(const RefPtr<MediaDevice>& aOther, const nsString& aID,
               const nsString& aGroupID, const nsString& aRawID,
-              const nsString& aName);
+              const nsString& aRawGroupID, const nsString& aName);
 
   uint32_t GetBestFitnessDistance(
       const nsTArray<const NormalizedConstraintSet*>& aConstraintSets,
@@ -127,6 +128,7 @@
   const nsString mID;
   const nsString mGroupID;
   const nsString mRawID;
+  const nsString mRawGroupID;
   const nsString mRawName;
 };
 